framework/atoms/buttons/buttons.html

```diff
@@ -1,4 +1,4 @@
-<a href="#" class="button button--{{ modifier }}">Link button</a>
-<a href="http://example.com" class="button button--{{ modifier }}">External link button</a>
-<button class="button button--{{ modifier }}">Button element</button>
-<input type="button" class="button button--{{ modifier }}" value="input[type='button']">
+<a href="#" class="btn btn-{{ modifier }}">Link btn</a>
```

Comment:
I think "External link button" is ok, wouldn't change it to "External link btn", as well as "Button element" rather than "btn element".